.github/workflows/run-tests.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
         cache: 'pip'
+        allow-prereleases: true
```

Comment:
Do we still need this or can it be removed now that `pytest-httpbin` is fully released?

Reply:
This is needed for 3.13rc